discord/discordDeleteMessage.html

```diff
@@ -0,0 +1,55 @@
+<script type="text/javascript">
+  RED.nodes.registerType('discordDeleteMessage', {
+    category: 'discord',
+    color: '#7289da',
+    defaults: {
+      name: {
+        value: "",
+        required: false
+      },
+      token: {
+        value: "",
+        required: true,
+        type: "discord-token"
+      }
+    },
+    inputs: 1,
+    outputs: 0,
+    icon: "discord.png",
+    label: function () {
+      return this.name || "discordDeleteMessage";
+    }
+  });
+</script>
+<script type="text/x-red" data-template-name="discordDeleteMessage">
+  <div class="form-row">
+    <label for="node-input-token"><i class="icon-tag"></i> token</label>
+    <input type="text" id="node-input-token" placeholder="token">
+  </div>
+  <div class="form-row">
+    <label for="node-input-name"><i class="icon-tag"></i> Name</label>
+    <input type="text" id="node-input-name" placeholder="Name">
+  </div>
+</script>
+<script type="text/x-red" data-help-name="discordDeleteMessage">
+  <p>Allows deleting of discord messages by supplying the message and channel id's.</p>
+
+  <h3>Inputs</h3>
+  <dl class="message-properties">
+    <dt>id
+        <span class="property-type">Object or string</span>
+    </dt>
+    <dd>the message Object or message id</dd>
+    <dt>channel
+      <span class="property-type">Object or string</span>
+    </dt>
+    <dd>the channel Object or channel id that the message is in</dd>
+    <dt class="optional">timedelay <span class="property-type">number</span></dt>
+    <dd>the time in milliseconds to wait before deleting the message, default is 0</dd>
```

Comment:
Node-red itself provides the `delay` node for this. Is that node sufficient? If not, why should delay functionality instead be implemented in `discordDeleteMessage`?

Reply:
Discord.js v13 also no longer supports this (see [the v12 -> v13 upgrade documentation entry for this](https://discordjs.guide/additional-info/changes-in-v13.html#message-delete)) so I recommend deleting this option.